### ISSUE TITLE
Xupload 5.0.1 not working with Yii 1.16.1

### ISSUE THREAD

**Issue by ifocus22** [Asgaroth/xupload#27]
Thanks for this great extension. I'm using it properly with Yii framework v 1.15.

I tried the same code with 1.16.1 but it's failing. The widget is displayed properly but the upload does not work. No info in tha app log, no error in the js console. I'm using the basic scenario : 

   public function actions()
    {
        return array(
            'upload'=>array(
                'class'            =>'xupload.actions.XUploadAction',
                'path'             =>Yii::app() -> getBasePath() . "/../uploads",
                'publicPath'   => Yii::app() -> getBaseUrl() . "/uploads",
            ),
        );
    }

    public function actionIndex()
    {

      Yii::import("xupload.models.XUploadForm");
        $model = new XUploadForm;
        $this -> render('index', array('model' => $model, ));
    }

**Comment by gplaza** [comment]
Same thing here (not working with yii 1.1.16) !
But i have an javascript error :

Error jquery :
TypeError: that is undefined
options = that.options,

The object "options" seem empty

**Comment by dev3k** [comment]
try changing the code in assets/jquery.fileupload-ui.js 

from 
var that = $(this).data('fileupload');
to 
var that = $("#fileupload").data('blueimpUI-fileupload');

http://stackoverflow.com/questions/23762825/why-is-this-datamycontrol-undefined

**Comment by Asgaroth** [comment]
Looks like Yii 1.1.16 updated the included version of jquery, so the correct solution would be to update the corresponding jquery upload library included in this extension.

I am no longer supporting this extension, so I am looking for any interested maintainers.

Please submit a PR, not sure how to transfer the access to updating the extension in the Yii site, probably just remove the downloads there, and point to this repository would work.

**Comment by gplaza** [comment]
@dev3k : This change doesn't work :(
same problem with Yii 1.1.16

**Comment by dev3k** [comment]
@gplaza you have to replace all the recurrences, try my working version of the file https://jsfiddle.net/khalilmajdalawi/os2gtamt/

**Comment by gplaza** [comment]
@dev3k : 
Sorry you are right !
I have changed this :
var that = $(this).data('fileupload');
to 
var that = $("#fileupload").data('blueimpUI-fileupload');

but in reality it's just the sample of stackoverflow .... (#fileupload) 

the good change is :
var that = $(this).data('blueimpUI-fileupload');

**Comment by Asgaroth** [comment]
Is it possible that one of you can create a PR? I can update the extension page in the yii site.

**Comment by gplaza** [comment]
Here a PR : https://github.com/Asgaroth/xupload/pull/28
thx for you extension Asgaroth :D

**Comment by Asgaroth** [comment]
Does it work if you replace `assets/jquery.fileupload-ui.js` with the [latest version of the jquery plugin](https://github.com/blueimp/jQuery-File-Upload) ?

**Comment by gplaza** [comment]
@Asgaroth : no, there is lot of refactoring to create if you update with the last version of Jquery Plugin.
for a simple fix is more simple to only change jquery.fileupload-ui.js.

ps : I'm not an expert in javascript but i have tried it.
